app/src/main/java/com/yapp/twix/service/TwixFirebaseMessagingService.kt

```
@@ -0,0 +1,106 @@
+            .from(this)
+            .notify(stableId, notification)
+    }
+
+    // PendingIntent의 requestCode에 활용될 안전한 notificationId 변환 메서드
+    private fun PushPayload.resolveStableNotificationId(): Int {
+        val fromDeepLink =
+            deepLink
+                ?.let {
+                    try {
+                        it.toUri()
+                    } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)